docs/input_data.md

```diff
@@ -304,3 +304,63 @@ your application. Use
 (CPU) shared memory or
 [`--shared-memory=cuda`](cli.md#--shared-memorynonesystemcuda) to use CUDA
 shared memory.
+
+## Dynamic gRPC
+
+When using the dynamic gRPC service kind,
+you can configure multiple input data via the `ipc_stream` field
+that can be used to generate a pool of input data (e.g. Protobuf messages) for the gRPC service.
+The Protobuf messages generated from each command specified in the `ipc_stream` field
+can be used to send individual inference requests to the gRPC service,
+and Perf Analyzer will cycle through each input data in a round-robin manner
+when sending inference requests,
+which is similar to its default behavior when the ipc_stream field is not specified.
+
+For example, given the following input JSON:
+```json
+{
+  "data": [
+    {
+      "ipc_stream": "python3 example.py --arg1 value1 --arg2 value2"
+    },
+    {
+      "ipc_stream": "python3 example.py --arg1 value3 --arg2 value4"
+    }
+  ]
+}
+```
+In this configuration, Perf Analyzer will read the Protobuf messages from each of the two commands specified in the `ipc_stream` field,
+and create a pool of two separate input data that can be used to send inference requests.
+Each input data can be customized, as in the example above,
+and contain any user-specified Protobuf messages
+(as long as they conform to the gRPC service's expected definition).
+
+### JSON Schema
+
+Here's the JSON schema for the input JSON file when using the dynamic gRPC service kind:
+```json
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Dynamic gRPC input JSON Schema",
+  "type": "object",
+  "properties": {
+    "data": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "ipc_stream": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "ipc_stream"
+        ]
+      }
+    }
+  },
+  "required": [
+    "data"
+  ]
+}
+```
```

Comment:
I might have missed the reason for this--is it just to show users how to use this feature?

The documentation with a simple example of `python3 input_stream.py` seems clear enough.

Reply:
@nicolasnoble asked for json schema (unless I misunderstood his ask).

Reply:
Yeah I mean I guess regardless of who asked for it, I'm just trying to understand the value-add.

Reply:
I think it's possible that our input JSON usage seems confusing for anyone who's not familiar with PA. So the value add for providing schema for dynamic grpc case would be to help the users see the full spec of the JSON file and remove any question marks. At least, that's why I think @nicolasnoble asked for it (correct me if I am reading your mind wrong).